main.rb

```diff
@@ -0,0 +1,141 @@
+require './person'
+require './book'
```

Comment:
- [ ] good job on the new methods 👍 however when running `ruby main.rb` to test the code the user gets an error because in `book.rb` line 14 the `end` keyword is missing.